spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -92,210 +92,19 @@
-          os_major_version: 9,
-          ref_id_suffix: 'bar',
-          supports_minors: [8]
-        )
-      end
-      let(:item) do
-        FactoryBot.create(
-          :v2_tailoring,
-          :with_mixed_rules,
-          :with_tailored_values,
-          policy: parent,
-          profile: canonical_profile,
-          os_minor_version: 8
-        )
-      end
-
-      it 'returns XCCDF tailoring file' do
-        get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
-
-        tailoring_values = V2::Tailoring.find(extra_params[:id]).value_overrides.keys
-        tailoring_file = Nokogiri::XML(response.body).remove_namespaces!
-        tailored_values = tailoring_file.xpath('//Profile/set-value/@idref')
-        selected, = tailoring_file.xpath('//Profile/select')
-                                  .partition { |sel| sel.attributes['selected'].value == 'true' }
-        rules_and_groups = selected.map { |r| r.attributes['idref'].value }
-        rules_only = rules_and_groups.select { |r| r.exclude?('content_rule_group') }
-
-        expect(response).to have_http_status :ok
-        expect(response.headers['Content-Type']).to eq('application/xml')
-        expect(tailoring_values).to eq(tailored_values.map(&:value))
-        expect(rules_only).to match_array(item.rules_added.map(&:ref_id))
-        expect(rules_and_groups - rules_only).to match_array(item.rule_group_ref_ids)
-      end
-    end
+    let(:account) { FactoryBot.create(:v2_account, :cert_authorized) }
+    let(:current_user) { FactoryBot.create(:v2_user, account: account) }
 
-    context 'with unauthorized policy' do
-      let(:canonical_profile) do
-        FactoryBot.create(:v2_profile)
-      end
-      let(:extra_params) do
-        {
-          # policy of a foreign account
-          policy_id: FactoryBot.create(:v2_policy, account: FactoryBot.create(:v2_account)),
-          id: item.id
-        }
-      end
-      let(:item) do
-        FactoryBot.create(
-          :v2_tailoring,
-          policy: parent,
-          profile: canonical_profile,
-          os_minor_version: 8
-        )
-      end
-
-      it 'results in 404 error' do
-        get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
-
-        expect(response).to have_http_status :not_found
-      end
-    end
-
-    context 'with mismatching set of IDs' do
-      let(:canonical_profile) do
-        FactoryBot.create(:v2_profile)
-      end
-      let(:extra_params) do
-        {
-          policy_id: FactoryBot.create(:v2_policy, account: FactoryBot.create(:v2_account)),
-          id: Faker::Internet.uuid
-        }
-      end
-
-      it 'results in 404 error' do
-        get :tailoring_file, params: extra_params.merge(parents: [:policy], format: :xml)
+    it_behaves_like 'tailoring_file'
+  end
 
-        expect(response).to have_http_status :not_found
-      end
-    end
+  context 'unauthorized via cert_auth' do
```

Comment:
I think it should be semantically:
```suggestion
  context 'authorized via regular authentication' do
```
But probably you don't need this context explicitly and can just unwrap the include.